front/app/hooks/useNavbarItems.ts

```diff
@@ -24,3 +26,21 @@ export default function useNavbarItems() {
 
   return navbarItems;
 }
+
```

Comment:
We truncate the titles coming back from the backend so that they're never longer than 20 characters (the `MAX_TITLE_LENGTH`). Most of the time this is not necessary (the default navbar items are always less than 20 characters, and the input field used to change their length is also limited to 20 characters) but in some languages like Greenlandish, or when you add a page with a really long title to the navbar, the limit can be exceeded. Ideally this should be something the backend takes care off- will add a Jira ticket for that